_contrib/new-topic

```diff
@@ -68,7 +60,17 @@ optech_mentions:
 # see_also:
 #   - title:
 #     link:
+
+## Optional extended summary.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
```

Comment:
It seems strange to me to have this comment before the end of the front matter section. Perhaps it can just be removed entirely.